package.json

```diff
@@ -201,28 +201,35 @@
     "vscode:prepublish": "tsc -p ./",
     "compile": "tsc -watch -p ./",
     "postinstall": "node ./node_modules/vscode/bin/install",
-    "tslint": "tslint -t verbose src/**/*.ts",
+    "tslint": "tslint -t verbose src/**/*.ts --exclude src/**/*.d.ts",
     "test": "node ./node_modules/vscode/bin/test"
   },
   "devDependencies": {
     "@types/dotenv": "^4.0.3",
     "@types/fs-extra": "^4.0.3",
     "@types/mocha": "^2.2.32",
     "@types/node": "^6.0.40",
+    "@types/request-promise": "^4.1.41",
+    "@types/strip-json-comments": "0.0.30",
+    "azure-arm-resource": "^3.1.1-preview",
     "mocha": "^2.3.3",
     "tslint": "^5.7.0",
     "typescript": "^2.0.3",
-    "vscode": "^1.1.16"
+    "vscode": "^1.1.17"
   },
   "dependencies": {
+    "azure-arm-containerregistry": "^2.2.0",
     "dotenv": "^5.0.1",
     "fs-extra": "^4.0.2",
     "is-port-reachable": "^2.0.0",
     "jsonc-parser": "^1.0.1",
+    "request": "^2.85.0",
+    "request-promise": "^4.2.2",
     "strip-json-comments": "^2.0.1",
     "vscode-extension-telemetry": "0.0.8"
   },
   "extensionDependencies": [
+    "ms-vscode.azure-account",
```

Comment:
azure account is already a dependency of azure-iot-toolkit. Depends on the parsing strategy of vscode, and we need keep an eye of the installation success rate

Reply:
Got it. I just want to make it more explicit as Edge extension now directly depends on azure-account.